examples/demo.js

```diff
@@ -240,7 +254,7 @@ window.onload = function () {
       'gifWidth': imgs[0].width,
       'gifHeight': imgs[0].height,
       'images': imgSrcs,
-      'frameDuration': 7,
+      'frameDuration': Math.abs(parseInt($('#js-download-gif-framerate').val())),
```

Comment:
Shouldn't this be framerate times the number of frames?

Reply:
If not, rename the id to frame duration

Reply:
I changed the id to frameDuration